oeps/processes/oep-0055/decisions/0001-use-backstage-to-support-maintainers.rst

```diff
@@ -0,0 +1,126 @@
+
+Status
+******
+
+**Draft**
+
+.. TODO Upon Acceptance
+
+	* Update OEP to mention the catalog-info.ymal file instead of the readme.
+		* Relevant link https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html#documenting-maintainers
+
+Context
+*******
+
+* :doc:`OEP-55 <../../oep-0055-proc-project-maintainers>` laid out that we should document the maintainers of a component.
+
+* As a maintainer it is hard to know what components you are maintaining.
+
+* As a community member it is currently difficult to quickly look up who the
+  maintainers are for multiple components quickly.
+
+* Given the number of repos that make up the Open edX platform and the
+  complexity of the job of maintaining them, we need a system for cataloging,
+  and making transparent the critical details of our repositories.
+
+Decision
+********
+
+To solve the above issues and to enable the potential to ease other
+discoverability and maintainership burdens, the Open edX community shall
+maintain a `backstage`_ instance and related metadata.
+
+Consequences
+************
+
+* To support the backstage instance ownership data will have to be stored in a
+  structured way.  We will follow Backstage best practices and store this data
+  in a ``catalog-info.yaml`` file in each component repository.
+
+* Once a backstage instance exists we have the opportunity to further leverage
+  it as a centralized access point for maintainers as well as community members
+  to quickly and easily assess different components and find relevant links to
+  code, documentation, owners, etc.
+
+* Backstage will also give us the ability to expose other health metrics to
+  maintainers so they can more easily assess the components they own.
+
+Rejected Alternatives
+*********************
+
+Spreadsheet
+===========
+
+Pros
+----
+
+* Easy to build and get a high level view of the system.
+
+Cons
+----
+
+* Not a beautiful UI for maintainers to work with.
+
+* Data overload as we put more and more data into it with little to no control
+  over the presentation of the data.
+
+Owners in the Readme
+====================
+
+Pros
+----
+
+* Easy to see while in the repo.
+
+Cons
+----
+
+* Harder to aggregate the data from the unstructured files in a way that would
+  allow us to easily build a `single pane of glass`_ where maintainers can see
+  all the things they own.
+
+References
+**********
+
+* `Backstage`_ project site.
+
+* `catalog-info.yaml`_ spec definition
+
+* Sample ``catalog-info.yaml`` file:
+
+  .. code:: yaml
+
+    apiVersion: backstage.io/v1alpha1
+    # (Required) Acceptable Values: Component, Resource, System
+    kind: Component
+    metadata:
+      name: 'docs.openedx.org'
+      description: "This repository contains the root documentation site for https://docs.openedx.org"
+      links:
+        - url: "https://github.com/openedx/docs.openedx.org"
+          title: "Source Code"
+          icon: "Code"
```

Comment:
The name, description, and source code URL as strike me as redundant with the repository's name, GitHub description, and GitHub URL. Is there anything we can do to de-dupe this or lint to keep it in sync?

Reply:
maybe it could be baked into logic for the templates we start from when making new files?

See also, placeholders in yaml: https://stackoverflow.com/questions/41620674/use-placeholders-in-yaml